<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Overview</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="catalogueLayout">
     <item>
      <widget class="QLabel" name="catalogue_label">
       <property name="text">
        <string>Catalogue:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="catalogue_dropdown">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="collections_group">
     <property name="title">
      <string>Collections</string>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="collectionsGroupLayout">
      <item>
       <layout class="QHBoxLayout" name="filterLayout">
        <item>
         <widget class="QLabel" name="filter_label">
          <property name="text">
           <string>Filter:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="filter_collections_input">
          <property name="placeholderText">
           <string>Type to filter collections...</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="view_details_btn">
          <property name="text">
           <string>View Details</string>
          </property>
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>View detailed metadata for the selected collection</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeView" name="collections_tree">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Click to select a collection. Double-click to view details.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="collections_count_label">
        <property name="text">
         <string>No collections loaded</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #666666; font-size: 11px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="selection_frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#selection_frame { background-color: #f0f4f8; border: 1px solid #d0d0d0; border-radius: 4px; padding: 8px; }</string>
     </property>
     <layout class="QHBoxLayout" name="selectionFrameLayout">
      <item>
       <widget class="QLabel" name="selected_label">
        <property name="text">
         <string>Selected:</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font-weight: bold;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="selected_collection_label">
        <property name="text">
         <string>None</string>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="item_count_label">
        <property name="text">
         <string></string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #666666;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
